tempto-ldap/src/main/java/com/teradata/tempto/internal/fulfillment/ldap/DefaultLdapObjectEntryManager.java

```diff
@@ -99,12 +102,22 @@ private String addLdapDefinition(LdapObjectDefinition ldapObjectDefinition)
 
         ldapObjectDefinition.getAttributes()
                 .forEach((k, v) -> entries.put(new BasicAttribute(k, v)));
+
+        List<ModificationItem> modificationItems = newArrayList();
+        for (String key: ldapObjectDefinition.getModificationAttributes().keySet())
```

Comment:
Is it possible to get rid of this `for` clause altogether and convert this too to streams API method call?